docs/UserGuide.md

```diff
@@ -14,6 +14,17 @@ guest management tasks done faster than traditional GUI apps.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Components of the UI
+
+![UiComponentLabels](images/UiComponentLabels.png)
```

Comment:
Should we change the image to one with a more appropriate Request rather than just "hi"?

Reply:
Agreed.